models/aeif_cond_beta_multisynapse.h

```diff
@@ -0,0 +1,418 @@
+     * accessible from the iteration function. The last four elements of this
+     * enum (DG_EXC, G_EXC, DG_INH, G_INH) will be repeated
+     * n times at the end of the state vector State_::y with n being the number
+     * of synapses.
+     */
+    enum StateVecElems
+    {
+      V_M = 0,
+      W,      // 1
+      DG_EXC, // 2
+      G_EXC,  // 3
+      DG_INH, // 4
+      G_INH,  // 5
+      STATE_VECTOR_MIN_SIZE
+    };
+
+    static const size_t NUMBER_OF_FIXED_STATES_ELEMENTS = 2; // V_M, W
+    // DG_EXC, G_EXC, DG_INH, G_INH
+    static const size_t NUMBER_OF_STATES_ELEMENTS_PER_RECEPTOR = 4;
+
+    std::vector< double_t > y_; //!< neuron state
+    int_t r_;                   //!< number of refractory steps remaining
+
+    State_( const Parameters_& ); //!< Default initialization
+    State_( const State_& );
+    State_& operator=( const State_& );
+
+    void get( DictionaryDatum& ) const;
+    void set( const DictionaryDatum& );
+
+  }; // State_
+
+  // ----------------------------------------------------------------
+
+  /**
+   * Buffers of the model.
+   */
+  struct Buffers_
+  {
+    Buffers_( aeif_cond_beta_multisynapse& );
+    Buffers_( const Buffers_&, aeif_cond_beta_multisynapse& );
+
+    //! Logger for all analog data
+    UniversalDataLogger< aeif_cond_beta_multisynapse > logger_;
+
+    /** buffers and sums up incoming spikes/currents */
+    std::vector< RingBuffer > spike_exc_;
+    std::vector< RingBuffer > spike_inh_;
+    RingBuffer currents_;
+
+    /** GSL ODE stuff */
+    gsl_odeiv_step* s_;    //!< stepping function
+    gsl_odeiv_control* c_; //!< adaptive stepsize control function
+    gsl_odeiv_evolve* e_;  //!< evolution function
+    gsl_odeiv_system sys_; //!< struct describing system
+
+    // IntergrationStep_ should be reset with the neuron on ResetNetwork,
+    // but remain unchanged during calibration. Since it is initialized with
+    // step_, and the resolution cannot change after nodes have been created,
+    // it is safe to place both here.
+    double_t step_;          //!< simulation step size in ms
+    double IntegrationStep_; //!< current integration time step,
+                             //!< updated by solver
+
+    /**
+     * Input current injected by CurrentEvent.
+     * This variable is used to transport the current applied into the
+     * _dynamics function computing the derivative of the state vector.
+     * It must be a part of Buffers_, since it is initialized once before
+     * the first simulation, but not modified before later Simulate calls.
+     */
+    double_t I_stim_;
+  };
+
+  // ----------------------------------------------------------------
+
+  /**
+   * Internal variables of the model.
+   */
+  struct Variables_
+  {
+
+    /** initial value to normalise excitatory synaptic conductance */
+    std::vector< double_t > g0_ex_;
+
+    /** initial value to normalise inhibitory synaptic conductance */
+    std::vector< double_t > g0_in_;
```

Comment:
This will always have exactly the same values as `g0_ex_`, so you really need only one `g0_` vector.